src/lib/common/ConfirmationModal.svelte

```diff
@@ -0,0 +1,27 @@
+<script lang="ts">
+    import Modal from "./Modal.svelte";
+    import { createEventDispatcher } from 'svelte';
```

Comment:
`npm run fmt` should change this to use double quotes. I'd vote for that for consistency for now. Happy to switch to something else, as long as I can trigger it from the command line somehow